core/src/main/java/org/apache/iceberg/actions/BinPackRewriteFilePlanner.java

```diff
@@ -19,11 +19,13 @@
 package org.apache.iceberg.actions;
 
 import java.io.IOException;
+import java.util.ArrayList;
 import java.util.List;
 import java.util.Map;
 import java.util.Set;
+import java.util.concurrent.atomic.AtomicInteger;
 import java.util.stream.Collectors;
-import java.util.stream.Stream;
+import org.apache.curator.shaded.com.google.common.collect.Lists;
```

Comment:
this seems off

Reply:
Thank you for pointing this out. I fixed this to the right Lists import

Reply:
This is wrong again

Reply:
Fixed this